content/en/docs/v1.0/tutorials/global-rate-limit.md

```diff
@@ -4,37 +4,37 @@ description:
 
-安装完成后，可以看到集群中增加了下面两个 NS，这两个 NS 中分别安装了基于 MetaProtocol 实现的 Dubbo 和 Thrift 协议的示例程序。
-你可以选用任何一个程序进行测试。
+Once the installation is complete, you can see the following two NSs have been added to the cluster,The two NSs have demo applications installed for Dubbo and Thrift protocols based on the MetaProtocol implementation respectively.
+You can choose either of them to test.
 
 ```bash
 ➜  ~ kubectl get ns|grep meta
 meta-dubbo        Active   16m
 meta-thrift       Active   16m
 ```
 
-## 什么是全局限流
+## What is global rate limiting?
 
-和[本地限流](/zh/docs/v1.0/tutorials/local-rate-limit/)不同的是，在使用全局限流时，所有的服务实例共用一个限流配额。全局限流通过一个限流服务器来实现在多个服务实例之间共享限流配额。在收到请求时，服务服务端的 Sidecar Proxy 会先向限流服务器发送一个限流查询请求，限流服务器在其自身的配置文件中读取限流的规则，根据规则判断一个限流请求是否触发了限流条件，然后将限流结果返回给 Sidecar Proxy。 Sidecar Proxy 根据该限流结果决定是禁止本次请求还是继续处理本次请求。
+Unlike [local rate limiting](/zh/docs/v1.0/tutorials/local-rate-limit/), when using a global rate limiting, all service instances share a single rate limiting quota. A global rate limiting server is used to ensure that. When receiving a request, Sidecar Proxy on the server side will first send a rate limiting query request to rate limiting server, which will read the rules in its own configuration file, then determine whether the request triggers a rate limiting condition according to the rules, finally return result to Sidecar Proxy. Based on the result of this rate limiting, Sidecar Proxy decides whether to disable this request or to continue processing it.
 
 ![](../global-rate-limit.png)
 
-## 在何时使用全局限流
-全局限流的特点是限流判断在限流服务器处统一进行的，因此不会受到服务实例数量的影响。但全局限流也会引入限流服务器这额外一跳，会带来附加的网络延迟。在大量客户请求的情况下，限流服务器自身是一个潜在的瓶颈点，部署和管理比本地限流更为复杂。
+## When to use global rate limiting?
+The feature of global rate limiting is that the limiting judgement is made uniformly at the rate limiting server and is therefore not affected by the number of service instances. However, global rate limiting also introduces an additional hop to the rate limiting server, which can introduce additional network latency. In the case of a large number of client requests, the rate limiting server itself is a potential bottleneck point and is more complex to deploy and manage than local limiting.
 
-如果用限流的目的是为了将服务实例的压力控制在合理的范围内，建议使用本地限流。因为本地限流是在每个服务实例的 Sidecar Proxy 处分别处理的，因此在这种场景下本地限流对服务实例的入向请求控制更为精确和合理。如果配合适当的 HPA 策略，在现有服务实例负载都打满的情况下还可以对服务实例进行水平扩容，保障服务的稳定运行。
+If the purpose of using rate limiting is to keep the pressure on the service instance within reasonable limits, it is recommended that local rate limiting be used. Because local rate limiting is handled separately at each service instance's Sidecar Proxy, local rate limiting is more precise and reasonable for controlling inbound requests to service instances in this scenario. With an appropriate HPA policy, it is also possible to horizontally scale the service instances when the existing service instances are fully loaded, ensuring stable service operation.
```

Comment:
建议：If you just want to keep the number of requests sent to a single service instance within a reasonable limit, you should use local rate limiting. That's because local rate limiting is handled separately at each service instance's Sidecar Proxy, local rate limiting is more precise and reasonable for controlling inbound requests to service instances in this scenario. Used along with an appropriate HPA policy trigger by rate limiting metrics, we can keep the server cluster healthy by horizontally scaling the server cluster when all the existing service instances reach their limiting quota .